src/apps/Permissions/Home/AllPermissions.js

```diff
@@ -23,8 +23,27 @@ function AllPermissions({
 }) {
   const [selectedEntityType, setSelectedEntityType] = useState(-1)
   const [searchTerms, setSearchTerms] = useState('')
+  const [page, setPage] = useState(0)
   const { layoutName } = useLayout()
 
+  const permissionsKey = permissions
+    .map(permission => `${permission.app.proxyAddress}-${permission.role.id}`)
+    .join('')
```

Comment:
we might as well use a `,` to join, to make it easier to debug this string if we need to :).